packages/terra-form-select/src/search/Frame.jsx

```diff
@@ -273,16 +291,23 @@ class Frame extends React.Component {
    */
   handleBlur(event) {
     // The check for dropdown.contains(activeElement) is necessary to prevent IE11 from closing dropdown on click of scrollbar in certain contexts.
-    if (this.dropdown && (this.dropdown === document.activeElement && this.dropdown.contains(document.activeElement))) {
+    const { isOpen } = this.state;
+    const { onBlur } = this.props;
+
+    // TODO: fixme
```

Comment:
Let's get this cleaned up

Reply:
working through the blur logic right now